WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -51,7 +51,11 @@ extension WooTab {
 
     // Note: currently only the Dashboard tab (My Store) view controller is set up in Main.storyboard.
     private static func visibleTabs(_ isHubMenuFeatureFlagOn: Bool) -> [WooTab] {
-        var tabs: [WooTab] = [.myStore, .orders, .products, .reviews]
+        var tabs: [WooTab] = [.myStore, .orders, .products]
+
+        if !isHubMenuFeatureFlagOn {
+            tabs.append(.reviews)
+        }
```

Comment:
nit: since there is another `if isHubMenuFeatureFlagOn { ... }` case, maybe this can be in the `else` case below?